app-crypt/mhash/mhash-0.9.9.9.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="Provides hash algorithms"
```

Comment:
I'm OK with this SUMMARY, but if anyone would like to suggest a variant, please do not hesitate to do so :)
What do you think of:
```
SUMMARY="A library of hash algorithms"
```
Hmm, I don't think this is better than your summary, after all...
Don't change your's unless someone finds something better.

Reply:
@fbrosson How about `SUMMARY="A uniform interface to hash algorithms"`?